BUILDING.md

```diff
@@ -30,3 +30,66 @@ v1.1.0
 ```
 
 Note: on Windows invoke the `gfsh.bat` script to print the version string.
+
+## Setting up IntelliJ
+
+The following steps have been tested with:
+
+* **IntelliJ IDEA 2018.2.2**
+
+1. Run `./gradlew --parallel devBuild` from Geode repository root to create compiler generated source
+
+2. Import project into IntelliJ IDEA
+
+   From the **Welcome to IntelliJ IDEA** window:
+
+   a. **Import Project ->** select *build.gradle* file from Geode repository root and press **Open**.
```

Comment:
On GitHub this doesn't render as list making reading it more challenging.  My understanding is that lettered lists are not valid markdown.  I might suggest using either unordered lists, numbers, or using html (since that is all markdown is anyway)

Reply:
I'll fix this, thanks

Reply:
In this case, stick with an ordered list. A set of nested numbers will be OK.

Reply:
Thanks Dave, that's what I settled on as well.  See my latest commit.